client/src/components/edit-client-page/index.jsx

```diff
@@ -0,0 +1,353 @@
+      </div>
+
+      <Paper>
+      <Box>
+      <Box paddingTop={2} paddingBottom={2} paddingLeft={2}>
+            <Grid container direction="row" alignItems="center">
+              <Grid item xs={10}>
+                <Typography variant="h5" align="left">
+                  Personal Information
+                </Typography>
+              </Grid>
+              <Grid item xs={1.6} align="right">
+                <EditIcon color="primary" />
+              </Grid>
+            </Grid>
+          </Box>
+          <Divider />
+
+          <Box paddingTop={2} paddingBottom={1} paddingLeft={3}>
+            <Grid container direction="row" alignItems="center">
+              <Grid item xs={4}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Name
+                </Typography>
+              </Grid>
+              <Grid item xs={8}>
+                <TextField
+                  type="text"
+                  id="firstName"
+                  name="firstName"
+                  value={editedName}
+                  onChange={handleNameChange}
+                />
+              </Grid>
+            </Grid>
+          </Box>
+          <Divider variant="middle" />
+
+          <Box paddingTop={2} paddingBottom={1} paddingLeft={3}>
+            <Grid container direction="row" alignItems="center">
+              <Grid item xs={4}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Email
+                </Typography>
+              </Grid>
+              <Grid item xs={8}>
+                <TextField
+                  type="text"
+                  id="firstName"
+                  name="firstName"
+                  value={editedEmail}
+                  onChange={handleEmailChange}
+                />
+              </Grid>
+            </Grid>
+          </Box>
+          <Divider variant="middle" />
+          <Box paddingTop={2} paddingBottom={1} paddingLeft={3}>
+          <Grid container direction="row" alignItems="center">
+            <Grid item xs={5}>
+              <Typography gutterBottom variant="body1" align="left">
+              Phone Number
+              </Typography>
+            </Grid>
+            <Grid item xs={6}>
+            <TextField
+              value={editedPhone}
+              onChange={handlePhoneChange}
+              id="phone"
+              name="phone"
+              InputProps={{
+                inputComponent: TextMaskCustom,
+              }}
+            />
+            </Grid>
+            <Grid item xs={1} id="info-card-icon">
+              <ContentCopyIcon />
+            </Grid>
+          </Grid>
+        </Box>
+          <Divider variant="middle" />
+          <Box paddingTop={2} paddingBottom={1} paddingLeft={3}>
+            <Grid container direction="row" alignItems="center">
+              <Grid item xs={4}>
+                <Typography gutterBottom variant="body1" align="left">
+                  Status
```

Comment:
The status should be one of "R and I", "Active" or "Closed". See https://github.com/uoftblueprint/tces/wiki/Database-Schema for more info on the potential values each field can take.